### PR TITLE
Add Natalie 0.1 - Swift script

### PR DESCRIPTION
**Pull request by krzyzanowskim**
Add formula for Natalie script

### CODE REVIEW

File: Library/Formula/natalie.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Natalie < Formula
+  homepage "https://github.com/krzyzanowskim/Natalie#readme"
+  url "https://github.com/krzyzanowskim/Natalie/archive/0.1.tar.gz"
+  version "0.1"
+  sha256 "943dab1b7ac16555ec3047983a3c397f7ebe7559d3c098c02a6ee6d8e966655e"
+
+  def install
+  	bin.install 'natalie.swift'
```
Comment (DomT4) [inline Library/Formula/natalie.rb]: Your indentation is off here. Use spaces rather than tabs to indent.

**Comment by DomT4** [comment]
Homebrew tends to [frown](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff) on self-submitted work, especially when the project is so new. One of the maintainers here in Homebrew/homebrew will make the call though.

**Comment by krzyzanowskim** [comment]
@DomT4 thx for pointing this out. I understand.

**Comment by mikemcquaid** [comment]
We'll pass on this for now, sorry!

**Pull request closed without merging**